### DISCUSSION TITLE
Mixxx 2.3. has been just released

### DISCUSSION THREAD

**Discussion by MadTiXx** [fayaaz/mixxx-pi-gen#27] [General]
Hi,
Just a quick notice that mixxx 2.3. has been recently released.

Could you do a new build of mixxx pi gen with this version?

Greetings
Thomas

**Comment by fayaaz** [comment]
You can always pick up a nightly from the github actions.

E.g. https://github.com/fayaaz/mixxx-pi-gen/actions/runs/999567603 under artifacts you can see the 'images' file with the latest 2.3 built and now keyfinder enabled.

I will have to test the new keyfinder changes and for some reason it looks like the image has increased 1gb in size - so  I will need to investigage that before another release.
